### ISSUE TITLE
25% signature generator failed with `signature failed to be produced: AssertionFailed("signature failed to verify")`

### ISSUE THREAD

**Issue by ppca** [sig-net/mpc#41]
[gcp log](https://console.cloud.google.com/logs/query;query=resource.type%3D%22gce_instance%22%0Ajson_payload.target%3D%22mpc_node::protocol::signature%22%0A%22signature%20failed%20to%20be%20produced%22;lfeCustomFields=jsonPayload%252Ftarget;cursorTimestamp=2024-11-06T16:49:47.163549870Z;startTime=2024-11-06T16:31:44.902Z;endTime=2024-11-06T17:31:44.902Z?project=near-cs-dev) showing `signature failed to be produced`, the error message is in the `err` field in jsonpayload.
this is part of investigating why dev signature success rate is <95%.
Each hour there are 50-60 signature generators failed because of this error.
Each hour there are 110-130 total signature generators failed. 
Each hour there are ~110-130 new signature generators started, and 100-130 restarted signature generators.
So basically `AssertionFailed("signature failed to verify")` constitutes 50% of all failed signature generators, and 25% of all signature generators.

**Comment by volovyks** [comment]
I believe this one is not relevant or a part of https://github.com/sig-net/mpc/issues/42
